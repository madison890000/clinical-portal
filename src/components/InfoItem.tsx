import { Typography } from '@mui/material';
import { capitalize } from '../utils';

export const InfoItem = ({
                             name,
                             value,
                         }: { name: string; value: string | number }) => {

    return (
        <div style={{
            display: 'flex',
            alignItems: 'center'
        }}>
            <div style={{
                height: 32,
                marginRight: 12
            }}>{capitalize(name)}:
            </div>
            <Typography variant="overline" display="block">
                {value}
            </Typography>
        </div>
    )
}

export default InfoItem
