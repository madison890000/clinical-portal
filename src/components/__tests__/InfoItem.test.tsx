import React from 'react';
import InfoItem from '../InfoItem';
import renderer from 'react-test-renderer';

test('renders InfoItem correctly', () => {
    const demoInfoItem = {
        name: 'Age',
        value: '18'
    };
    const InfoItemDom = renderer.create(<InfoItem {...demoInfoItem} />).toJSON();
    expect(InfoItemDom).toMatchSnapshot();
});
