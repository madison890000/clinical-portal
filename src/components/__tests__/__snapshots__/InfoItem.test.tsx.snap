// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InfoItem correctly 1`] = `
<div
  className="container"
>
  <div
    className="name"
  >
    Age
    :
  </div>
  <span
    className="MuiTypography-root MuiTypography-overline css-1kdyykn-MuiTypography-root"
  >
    18
  </span>
</div>
`;
